test/grpc/leaderboard_test.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:faker/faker.dart';
+import 'package:nakama/api.dart' as api;
+import 'package:nakama/nakama.dart';
+import 'package:test/expect.dart';
+import 'package:test/scaffolding.dart';
+
+import '../config.dart';
+
+void main() {
+  group('[gRPC] Test Leaderboard', () {
+    late final NakamaBaseClient client;
+    late final Session session;
+
+    setUpAll(() async {
+      client = getNakamaClient(
+        host: kTestHost,
+        ssl: false,
+        serverKey: kTestServerKey,
+      );
+
+      session = await client.authenticateDevice(deviceId: faker.guid.guid());
+    });
+
+    test('list leaderboard records', () async {
+      final result = await client.listLeaderboardRecords(
+          session: session, leaderboardId: 'test');
+
+      expect(result, isA<api.LeaderboardRecordList>());
+      expect(result.records, hasLength(0));
+    });
+  });
+}
```

Comment:
Yep, sounds good. Please add this lua script on the startup so that the tests succeed automatically. Maybe on the long run we can also add flutter tests via github actions, so this would be helpful to get the full test setup running and this is why it's very important to not have manual required actions.